Sources/Client/Views/Play/WithRenderCoordinator.swift

```diff
@@ -7,24 +7,41 @@ import DeltaRenderer
 struct WithRenderCoordinator<Content: View>: View {
   var client: Client
   var content: (RenderCoordinator) -> Content
+  var cancel: () -> Void
 
-  init(for client: Client, @ViewBuilder content: @escaping (RenderCoordinator) -> Content) {
+  init(for client: Client, @ViewBuilder content: @escaping (RenderCoordinator?) -> Content, cancellationHandler cancel: @escaping () -> Void) {
     self.client = client
     self.content = content
```

Comment:
How does this compile? The parameter `content` is a function taking an optional render coordinator but the property `content` is a function taking a required render coordinator.

Reply:
Good catch, I'm actually not sure why the compiler isn't complaining. Xcode for sure at least didn't even mention a  warning 😮

Reply:
should be alright now